features/management_students/createStudent.feature

```diff
@@ -10,12 +10,34 @@ Feature: Set of tests to create students
 		Given <userRole> logs in with email '<username>' and password '<password>'
 		And <userRole> goes to management
 		And <userRole> goes to students management
-		When <userRole> set student firstname '<firstName>', lastname '<secondName>', email '<studentEmail>'
+		When <userRole> set student firstname '<firstName>', lastname '<secondName>', email '<studentEmail>', birthday '<birthday>'
 		And <userRole> sees that student with email '<studentEmail>' is visible on the list
 		And <userRole> manually submits consent for user with e-mail '<studentEmail>', thus generates a random password for him
 		And <userRole> logs out
 		And student logs in with email '<studentEmail>' and password genarated by admin during manual submission of consent
 		Then student should see that data protection is already accepted and performs first login actions: password change '<newPasswordStudent>'
 		Examples:
-			| userRole | firstName | secondName | studentEmail              | username                     | password       | newPasswordStudent |
-			| admin    | Georg     | Georgmann  | georgmann@schul-cloud.org | kai.admin.qa@schul-cloud.org | Schulcloud1qa! | Schulcloud1!!      |
+			| userRole | firstName | secondName | studentEmail              | username                     | password       | newPasswordStudent | birthday   |
```

Comment:
Would prefer lastName instead of secondName, but we can change this later as well, if  it's the only thing that bothers someone.

Reply:
please change it, to lastName

Reply:
fixed